tests/qiskit_convert_test.py

```diff
@@ -1496,7 +1497,7 @@ def test_round_trip_with_qiskit_transpilation() -> None:
         basis_gates=["cx", "sx", "x", "rz", "if_else"],
         seed_transpiler=0,
     )
-    pass_manager = level_3_pass_manager(config)
+    pass_manager = level_2_pass_manager(config)
```

Comment:
I think the need for this change is not related to symbols, but to some stricter validation happening in the `PassManager` in 2.1.0, mentioned [here](https://quantum.cloud.ibm.com/docs/en/api/qiskit/release-notes/2.1#transpiler-upgrade-notes). It was hard for me to tell what was wrong with the code so I opted for this simple change which avoids the issue.